src/api/apps/sessions/model.js

```diff
@@ -68,12 +68,19 @@ export const save = (inputData, callback) => {
       _id: id,
     }
 
-    db.sessions.save(data, callback)
+    db.collection("sessions").updateOne(
+      { _id: data._id },
+      { $set: inputData },
+      { upsert: true },
```

Comment:
If someone with better knowledge can confirm that an `upsert` here is correct, it would be appreciated. Otherwise I will consider the passing tests enough validation.

Reply:
Not knowledgeable, but this seems reasonable.